File: Java/soma_vetores.java
import java.util.Scanner;

public class soma_vetores {
    public static void main(String[] arg){
        Scanner sc = new Scanner(System.in);
        int N;
        double soma = 0, media;
        System.out.print("digite quantos numeros deseja inserir no vetor : ");
        N = sc.nextInt();

        double[] vet = new double[N];

        for (int i=0;i<N;i++){
            System.out.print("digite o numero de posicao["+i+"]: ");
            vet[i] = sc.nextDouble();
            soma += vet[i];
        }
        media = soma/N;

        System.out.print("valores =");
        for (int i=0;i<N;i++){
            System.out.print(" "+vet[i]);
        }
        System.out.println("\nSoma = "+soma);
        System.out.println("Media = "+media);
        sc.close();
    }
    
}

File: Python/soma_vetores.py
N = int(input("Digite quantos numeros deseja digitar: "))
vet = [0 for x in range(N)]

soma = 0
for i in range(0, N):
    vet[i] = int(input(f"Digite o numero do elemento[{i}]: "))
    soma = soma + vet[i]

print(f"VALORES = ", end = "")
for i in range (0, N):
    print(f"{vet[i]} ", end = "")
print(f"\nSOMA = {soma}")
media = soma/N
print(f"MEDIA = {media}")
